prime-router/metadata/fhir_mapping/hl7/datatypes/CWE/Annotation.yml

```diff
@@ -0,0 +1,125 @@
+text:
+  type: STRING_ALL
+  valueOf: CWE
+  generateList: true
+  expressionType: HL7Spec
+
+extension_cwe1:
```

Comment:
So I encountered something similar for DLN[Identifier], but I took the approach of capturing the entire CWE in a codeable concept extension
```
extension:
  generateList: true
  expressionType: nested
  expressionsMap:
    url:
      type: SYSTEM_URL
      value: dln2-issuer
    valueCodeableConcept:
      valueOf: datatypes/CWE/CodeableConcept
      expressionType: resource
      specs: DLN.2
```

I think that would work here, but curious if you had thoughts on that